docs/connectors/configuration-parameters/rate-limiting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Rate Limiting"
+sidebar_position: 2
+---
+
+Destination connectors can be configured to limit the rate at which records can be written. This is especially useful when the destination resource has a rate limit to ensure that the connector does not exceed it. By default, Conduit does not limit the rate at which records are written.
+
+## Configuration parameters
+
+* `sdk.rate.perSecond`: Maximum number of records written per second (0 means no rate limit).
+* `sdk.rate.burst`: Allow bursts of at most X records (0 or less means that bursts are not limited). Only takes effect if a rate limit per second is set. Note that if `sdk.batch.size` is bigger than `sdk.rate.burst`, the effective batch size will be equal to `sdk.rate.burst`.
+
+## Example
+
+The pipeline will generate an infinite number of structured records in bursts during 5 seconds at a maximum rate, and write them to the log. When you run it, you'll notice that by using both `sdk.rate.perSecond` and `sdk.rate.burst`, the log destination connector will limit the rate of processed records to 1 burst per second.
+
+```yaml
+version: 2.2
+pipelines:
+  - id: generator-to-log
+    status: running
+    description: >
+      Example pipeline using the generator source connector and the log destination connector.
+      Showing how to limit the rate of processing records.
+    connectors:
+      - id: example
+        type: source
+        plugin: generator
+        settings:
+          rate: 0
+          recordCount: 0
+          burst.generateTime: 5s
+          burst.sleepTime: 0s
```

Comment:
Since these parameters are optional and they are not the point of this example, I would simply remove them and let the generator produce records as fast as possible, the destination will then rate limit the whole pipeline.

Reply:
@lovromazgon the idea behind adding those even though they are optional was to illustrate than even when provided, they rate takes effect.

Reply:
My worry is that it will just confuse people 😕 besides, currently the burst is not doing anything, since sleep time is `0s` it means that the generator will simply generate records all the time, so removing these params should no change the outcome.

Reply:
Particularly the word `burst` in this parameters makes me think we should not include it in the example, because these might be confused with `sdk.rate.burst`, which have no connection with each other.